components/composed/model/ModelListPage/ModelListPage.tsx

```diff
@@ -22,11 +25,10 @@ type ModelListPageProps<
   U extends PaginatedConnectionish,
   V extends Record<string, unknown> = Record<string, unknown>
 > = Omit<ModelListProps<T, U, V>, "view"> &
-  Pick<ModelListActionsProps, "viewOptions"> & {
+  Pick<ModelListActionsProps, "viewOptions"> &
+  Pick<HeaderProps, "headerStyle" | "hideHeader"> & {
```

Comment:
This looks right to me - we're extending a type here, while the other components use an interface. An interface is mutable, and we just want to add two new properties to what's already defined.

Here's more info on the difference between using `interface extends` and `type &`:
https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces